Resources/Locale/en-US/chat.ftl

```diff
@@ -0,0 +1,19 @@
+# chat manager
+chat-manager-ooc-chat-enabled-message = OOC chat has been enabled.
+chat-manager-ooc-chat-disabled-message = OOC chat has been disabled.
+chat-manager-admin-ooc-chat-enabled-message = Admin OOC chat has been enabled.
+chat-manager-admin-ooc-chat-disabled-message = Admin OOC chat has been disabled.
+chat-manager-max-message-length-exceeded-message = Your message exceeded {$limit} character limit
+chat-manager-no-headset-on-message = You don't have a headset on!
+chat-manager-server-wrap-message = SERVER: \{{0}}
```

Comment:
these little bastards were tricky. It has to be either \{{0}} OR we could change how message wraps are handled, but I didn't want to change it since there is too much stuff already in this PR

Reply:
ok, had to change these to {"{0}"}, but they work now nicely